app/views/items/dashboard.erb

```diff
@@ -1,18 +1,16 @@
 <!DOCTYPE html>
```

Comment:
This makes me think you didn't update the branch you were working on to the newest version of the master. All of these references to "Bazaar Crystals" were 2 PR's ago. I'm not going to close this pull request until we compare face to face in the morning.